chapters/tutorials/index.rst

```diff
@@ -0,0 +1,403 @@
+==================================
+Tutorial 1
+==================================
+
+.. contents:: Table of Contents
+    :depth: 3
+
+
+We will use th `hello world chatbot <https://github.com/jina-ai/jina#run-quick-demo>`_ for this tutorial. You can find the complete code `here <https://github.com/jina-ai/jina/tree/master/jina/helloworld/chatbot>`_ and we will go step by step.
+
+At the end of this tutorial, you will have your own chatbot. You will use text as an input and get a text result as output.
+For this example, we will use a `covid dataset <https://www.kaggle.com/xhlulu/covidqa>`_.
+You will understand how every part of this example works and how you could create new apps with different datasets on your own.
+
+Set-up & overview
+----------------------------------
+
+We recommend creating a `new python virtual environment <https://docs.python.org/3/tutorial/venv.html>`_, to have a clean install of Jina and prevent dependency clashing.
+
+We can start by installing Jina:
+
+  .. code-block:: python
+
+    pip install jina
+
+For more information on installing Jina, refer to this `page <https://docs.jina.ai/chapters/install/os/via-pip>`_.
+
+And we need the following dependencies:
+
+  .. code-block:: python
+
+    pip install click==7.1.2
+    pip install transformers==4.1.1
+    pip install torch==1.7.1
+
+
+Once you have Jina and the dependencies installed, let's take a broad overview of what we should do:
+
+.. image:: res/flow.png
+   :width: 600
+
+At the beginning of the Flow, you have your data, and this can be any type:
+
+* Audio
+* Image
+* Video
+* Text
```

Comment:
Let's order this by frequency of use/complexity (according to our survey results):

- Text
- Image**s**
- Audio
- Video